apps/prairielearn/src/pages/instructorCourseAdminOnboarding/instructorCourseAdminOnboarding.html.ts

```diff
@@ -0,0 +1,90 @@
+import { html } from '@prairielearn/html';
+
+import { HeadContents } from '../../components/HeadContents.html.js';
+import { Navbar } from '../../components/Navbar.html.js';
+import { CourseSyncErrorsAndWarnings } from '../../components/SyncErrorsAndWarnings.html.js';
+import type { OnboardingStepInfo } from '../../lib/onboarding.js';
+
+export function InstructorCourseAdminOnboarding({
+  steps,
+  resLocals,
+}: {
+  steps: OnboardingStepInfo[];
+  resLocals: Record<string, any>;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals, pageTitle: 'Onboarding checklist' })}
+      </head>
+      <body>
+        ${Navbar({ resLocals })}
+        <main id="content" class="container">
+          ${CourseSyncErrorsAndWarnings({
+            authz_data: resLocals.authz_data,
+            course: resLocals.course,
+            urlPrefix: resLocals.urlPrefix,
+          })}
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              <h1>Onboarding checklist</h1>
+            </div>
+            <div class="card-body text-left" style="text-wrap: balance;">
```

Comment:
Nit, we shouldn't need these styles on this page (I recognize these from the empty states, where they were necessary 🙂)

```suggestion
            <div class="card-body">
```